domain/schema/model/sql/0020-unit.sql

```diff
@@ -67,6 +67,34 @@ CREATE TABLE unit_agent_version (
     REFERENCES architecture (id)
 );
 
+-- v_unit_agent_version procides a convenience view on the unit_agent_version
+-- table reporting the architecture name as well as the id.
+CREATE VIEW v_unit_agent_version AS
```

Comment:
We only use this view once. Merge it with the query in `GetUnitRunningAgentBinaryVersion`.

Reply:
Removed